woocommerce/woocommerce
# Launch your store: Address status issues and filesystem

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR addresses some left over bits from https://github.com/woocommerce/woocommerce/pull/45606.

* The status chip now only shows on the homescreen
* The "Customize 'Coming soon' page" link is now working
* Refactor competing folders `/launch-your-store` and `/launch-store` by placing the hub in `plugins/woocommerce-admin/client/launch-your-store/hub`



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Fire up JN site
2. Turn on the feature flag Tools > WCA Test Helper > Features > launch-your-store
3. See the status chip only appear on the homescreen
4. Click the "Customize 'Coming soon' page" link and see that it goes to the address described in https://github.com/woocommerce/woocommerce/pull/45548



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Update unreleased feature

</details>
